### PR TITLE
Part5

### PR DESCRIPTION
**Pull request by GinaAntonini**

### CODE REVIEW

**Comment by morecallan** [comment]
👉 poke @GinaAntonini

File: javascripts/controllers/EditCtrl.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ app.controller("EditCtrl", function($location, $routeParams, $scope, $rootScope,
     getContact();
 
     $scope.updateContactInFirebase = (contact, contacts) => {
```
Comment (zoeames) [inline javascripts/controllers/EditCtrl.js]: why are you passing contacts in here?  do you need contact and contacts

File: javascripts/services/NewContactService.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ app.service("NewContactService", function($http, $q, FIREBASE_CONFIG){
     return $q((resolve, reject) => {
       $http.get(`${FIREBASE_CONFIG.databaseURL}/contacts.json?orderBy="uid"&equalTo="${userUid}"`).then((results) => {
         let fbContacts = results.data;
+        // console.log("fbcontacts", fbContacts);
```
Comment (zoeames) [inline javascripts/services/NewContactService.js]: delete this

File: javascripts/services/NewContactService.js [hunk 3]
```diff
@@ -38,17 +39,14 @@ app.service("NewContactService", function($http, $q, FIREBASE_CONFIG){
 
 
 	const postNewContact = (newContact) => {
+    console.log(newContact);
```
Comment (zoeames) [inline javascripts/services/NewContactService.js]: delete this console

File: partials/favorites.html [hunk 4]
```diff
@@ -7,14 +7,15 @@ <h1>Favorites</h1>
...
-				<p class="streetAddress"><strong>Address:</strong> {{favorite.streetAddress}}</p>
+			   <p class="firstName"><strong>Name: {{favorite.firstName}} {{favorite.lastName}}</p>
+				<p class="streetAddress">Address: {{favorite.streetAddress}}</p>
 				<p class="city">City: {{favorite.city}}</p>
 				<p class="state">State: {{favorite.state}}</p>
 				<p class="zip">Zip: {{favorite.zip}}</p>
 				<p class="phoneNumber">Phone Number: {{favorite.phoneNumber}}</p>
 				<p class="email">Email: {{favorite.email}}</p>
 				<p class="birthday">Birthday: {{favorite.birthday}}</p>
+				<span><i ng-class="{'fa fa-paw fa-3x': favorite.hasPet, '': !favorite.hasPet}"></i></span>
```
Comment (zoeames) [inline partials/favorites.html:18]: very nice use of ng-class!

File: javascripts/controllers/EditCtrl.js (hunk 1 above)
Comment (GinaAntonini) [reply javascripts/controllers/EditCtrl.js]: I need contact. I deleted contacts!

File: javascripts/services/NewContactService.js (hunk 2 above)
Comment (GinaAntonini) [reply javascripts/services/NewContactService.js]: done

File: javascripts/services/NewContactService.js (hunk 3 above)
Comment (GinaAntonini) [reply javascripts/services/NewContactService.js]: done

File: partials/favorites.html (hunk 4 above)
Comment (GinaAntonini) [reply partials/favorites.html:18]: :grin:

**Comment by zoeames** [comment]
awesome 👍

**Pull request merged**